pkg/apis/extensions/v1alpha1/types_operatingsystemconfig.go

```diff
@@ -227,12 +227,25 @@ const (
 	OperatingSystemConfigSecretDataKey = "cloud_config" // #nosec G101 -- No credential.
 )
 
+// CgroupDriverName is a string denoting the CRI cgroup driver.
+type CgroupDriverName string
+
+const (
+	// CgroupDriverCgroupfs sets the cgroup driver to cgroupfs.
+	CgroupDriverCgroupfs CgroupDriverName = "cgroupfs"
+	// CgroupDriverSystemd sets the cgroup driver to system.
+	CgroupDriverSystemd CgroupDriverName = "systemd"
```

Comment:
Nit: I am not happy with the wording of the documentation strings because a constant cannot actively set something. It may be used to do it, but itself it will never set it.